tide_core.module

```diff
@@ -448,3 +448,27 @@ function tide_core_system_breadcrumb_alter(Breadcrumb &$breadcrumb, RouteMatchIn
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function tide_core_node_presave(NodeInterface $node) {
+  $node_state = $node->get('moderation_state')->getValue()[0]['value'];
+
+  if ($node_state === 'published') {
+    $log = [
+      'type' => 'node',
+      'operation' => 'published',
+      'description' => t('%type: %title - New node created with workflow state %new_state', [
```

Comment:
Is it necessarily a new node though? I would check this language with the product owner as they don't really talk in terms of nodes.

Reply:
@anthony-malkoun Will check with the product owner.

Reply:
Hey @anthony-malkoun,

Checked with Nick. About the term 'node'.

He requested the use of the term 'page' for better readability.

Changes are made to reflect this.